src/main/java/cms/gov/madie/measure/services/MeasureLockService.java

```diff
@@ -31,26 +35,47 @@ public synchronized LockResponse lockMeasure(String measureId, String userName)
 
     try {
       measureLockRepository.insert(lock);
-      return new LockResponse(true, userName); // not locked by someone else
+      return new LockInfo(true, userName, measureId); // not locked by someone else
     } catch (DuplicateKeyException ex) {
       Optional<MeasureLock> existingLock = measureLockRepository.findByMeasureId(measureId);
       if (existingLock.isPresent()) {
         String lockedBy = existingLock.get().getLockedBy();
         boolean locked = !lockedBy.equals(userName);
-        return new LockResponse(locked, lockedBy);
+        return new LockInfo(locked, lockedBy, measureId);
       }
-      return new LockResponse(true, null); // fallback
+      return new LockInfo(true, null, measureId); // fallback
     }
   }
 
-  public synchronized LockResponse unlockMeasure(String measureId, String userName) {
+  public synchronized LockInfo unlockMeasure(String measureId, String userName) {
     Optional<MeasureLock> existingLock = measureLockRepository.findByMeasureId(measureId);
     // it's our lock. We delete it.
     if (existingLock.isPresent() && existingLock.get().getLockedBy().equals(userName)) {
       measureLockRepository.deleteByMeasureId(measureId);
-      return new LockResponse(false, null); // Successfully unlocked
+      return new LockInfo(false, null, measureId); // Successfully unlocked
     }
     // it's not our lock. we dont do anything
-    return new LockResponse(true, existingLock.map(MeasureLock::getLockedBy).orElse(null));
+    return new LockInfo(true, existingLock.map(MeasureLock::getLockedBy).orElse(null), measureId);
+  }
+
+  public synchronized List<String> unlockByUser(String userName) {
+    List<String> deleteMesssages = new ArrayList<>();
+    deleteMesssages.add("Delete measure locks for harpId: " + userName);
+    List<MeasureLock> existingLocks = measureLockRepository.findAllByLockedBy(userName);
+    log.info(
+        (CollectionUtils.isNotEmpty(existingLocks) ? existingLocks.size() : "No")
+            + " measure locks found for harpId: "
+            + userName);
+    if (CollectionUtils.isNotEmpty(existingLocks)) {
+      existingLocks.stream()
```

Comment:
Stream API is not needed, I guess most of the things are similar to cql-library-service PR. Synchronized is not needed.